{
	"id": "CVE-2020-13254",
	"tags": [
		{
			"label": "Product",
			"message": "n/a"
		},
		{
			"label": "Version",
			"message": "n/a"
		},
		{
			"label": "Vulnerability",
			"message": "n/a"
		}
	],
	"description": " An issue was discovered in Django 2.2 before 2.2.13 and 3.0 before 3.0.7. In cases where a memcached backend does not perform key validation, passing malformed cache keys could result in a key collision, and potential data leakage.",
	"poc": {
		"references": [
			"https://www.oracle.com/security-alerts/cpujan2021.html"
		],
		"githubs": [
			"https://github.com/0xT11/CVE-POC",
			"https://github.com/ARPSyndicate/cvemon",
			"https://github.com/Live-Hack-CVE/CVE-2020-13254",
			"https://github.com/Qubo-FNSD/Mapl-App-NVDs",
			"https://github.com/danpalmer/django-cve-2020-13254",
			"https://github.com/developer3000S/PoC-in-GitHub",
			"https://github.com/hectorgie/PoC-in-GitHub",
			"https://github.com/nomi-sec/PoC-in-GitHub",
			"https://github.com/soosmile/POC"
		]
	}
}
